lms/templates/instructor/instructor_dashboard_2/cohort-discussions-inline.underscore

```diff
@@ -0,0 +1,30 @@
+<hr class="divider divider-lv1" />
+
+<form action="" method="post" id="cohort-inline-discussions-form" class="cohort-inline-discussions-form">
+    <div class="wrapper cohort-management-supplemental">
+        <div class="form-fields">
+            <div class="form-field">
+                <div class="inline-discussion-topics">
+                    <h3 class="subsection-title"><%- gettext('Content-Specific Discussion Topics') %></h3>
+                    <p><%- gettext('Specify whether content-specific discussion topics are divided by cohort.') %></p>
+                    <div class="always_cohort_inline_discussions">
+                        <input type="radio" name="inline" class="check-all-inline-discussions" <%- alwaysCohortInlineDiscussions ? 'checked="checked"' : '' %>/>
+                        <span class="all-inline-discussions"><%= interpolate('Always cohort content-specific discussion topics') %></span>
```

Comment:
@symbolist Originally, we wanted to reflect exactly the same name as the flag was named in Advanced Settings so that users would understand that the behavior of that control had not changed. The word "Always" was meaningful in that it highlighted that future as well as existing inline discussion topics were affected by this setting. It's true that "Cohort all... topics" would be more parallel to "Cohort some...topics". Perhaps that wording does adequately capture that all content-specific discussion topics created after selecting this option are also cohorted. @explorerleslie @marcotuts what are your thoughts? If you all think that we don't lose nuances by moving away from "Always cohort..." then I agree with the change.